makeup-example/Makefile

```diff
@@ -1,100 +1,65 @@
-logs:
+logs: ## Follow docker logs from the containers
 	docker-compose logs -f --tail=50
 
-# Show current state of the environment.
-status:
+status: ## Show status of the environment
 	docker-compose ps
 	docker-compose top
 
+# Use the current list of plantuml files to define a list of required pngs.
+diagrams = $(patsubst %.plantuml,%.png,$(wildcard docs/diagrams/*.plantuml))
 
-# =============================================================================
-# OTHER COMMAND TARGETS
-# =============================================================================
-# These commands are specific to this project.
-
-# Check if the environment meets requirements. Requires containers are up.
-validate: ensure-php
-	# Example
-	docker-compose exec php drush --version
-
-# Reinstall Drupal if required (normally not).
-drupal-reinstall: ensure-php
-	docker-compose exec php drush site-install -y minimal --config-dir=../config/sync --account-mail=noreply+proref@reload.dk --account-pass=admin --site-mail=noreply+proref@reload.dk --site-name=ProRef --db-su=root --db-su-pw=root --db-url=mysql://db:db@db/db
-
-# Import the local config into a running Drupal site.
-drupal-locale-update: ensure-php
-	docker-compose exec php drush -y locale-check
-	docker-compose exec php drush -y locale-update
-
-# Run Drupal Console with an arbitrary set of parameters.
-# 	Example: make drupal-run CMD=site:status
-drupal-run: ensure-php
-	docker-compose exec php drupal ${CMD}
-
-# Run Drush with an arbitrary set of parameters.
-# 	Example: make drush-run CMD=status
-drush-run: ensure-php
-	docker-compose exec php drush ${CMD}
+# The default docs target depends on all png-files
+docs: $(diagrams) ## Build plantuml-diagrams for the documentation
 
+# Static pattern that maps between diagram pngs and plantuml-files.
+$(diagrams): docs/diagrams/%.png : docs/diagrams/%.plantuml
+	@echo '$< -> $@'
+	rm -f $@
+	cat $< | docker run --rm -i think/plantuml -tpng > $@
 
 # =============================================================================
 # HELPERS
 # =============================================================================
 # These targets are usually not run manually.
 
 # Fetch and replace updated containers and db-dump images and run composer install.
-update: docker-pull
-	docker-compose rm --stop --force
-	docker-compose up -d php
-	# wait just a bit before proceeding, since the php container is not ready right away.
-	sleep 5s
-	docker-compose exec php /bin/bash -c 'cd .. && rm -rf vendor/'
-	docker-compose exec php /bin/bash -c 'cd .. && composer --no-interaction install --optimize-autoloader'
-	docker-compose exec php drush --version
-	#docker-compose exec php drush cache-rebuild
-	#docker-compose exec php drush -y updatedb --entity-updates
-	#docker-compose exec php drush -y config-import
-
-ensure-php:
-	docker-compose up -d --no-deps php
+_reset-container-state: _docker-pull
+	./scripts/docker/docker-reset.sh
+
+_ensure-php:
```

Comment:
Is this ever called? Looks like we could just do without it now that we don't have any of the more specialised tasks.

Reply:
Fixed